### PR TITLE
Move run-kinto to get-started

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Moved run Kinto to get started
* [x] Move system requirements installation to run Python section
* [x] Move postgresql instruction to running in production
* [x] Added a very basic Javacscript tutorial page

### CODE REVIEW

**Comment by leplatrem** [comment]
* Run kinto is not hidden in the tutorials section
* Get started shows how to run and points to tutorials
* Tutorials now mention API in title

File: docs/tutorials/javascript.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
+.. _tutorial-javascript:
+
+Kinto for JavaScript applications
+#################################
+
+Kinto is particularly relevant for the Web, thus it is very probable that you'll
+want to interact with the server using JavaScript.
+
+This tutorial presents the basics, but you can look at
```
Comment (almet) [inline docs/tutorials/javascript.rst]: you can *also* look at

File: docs/tutorials/javascript.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+:ref:`the applications examples <app-examples>`, most of them use JavaScript.
+
+
+Offline and synchronization
+---------------------------
+
+We built :github:`kinto.js <Kinto/kinto.js>`, a JavaScript client dedicated
+to synchronizing remote records with a local database.
```
Comment (almet) [inline docs/tutorials/javascript.rst]: synchronising?

File: docs/tutorials/javascript.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+The best way to start is :rtd:`to follow the dedicated tutorial <kintojs>` !
```
Comment (almet) [inline docs/tutorials/javascript.rst]: This doesn't link to the tutorial, the link can be more precise.

File: docs/tutorials/javascript.rst (hunk 2 above)
Comment (almet) [inline docs/tutorials/javascript.rst]: Consider adding `that's what we call "offline first": the data is also available locally, and can be synced when a connection is available`.

File: docs/tutorials/javascript.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    fetch(bucketURL, putOptions)
+      .then(function (response) {
+        return fetch(collectionURL, putOptions);
+       });
+
+Now, we are able to create some records:
+
+.. code-block:: javascript
+
+    var data = {title: "How Kinto changed my life", date: new Date().toISOString()};
```
Comment (almet) [inline docs/tutorials/javascript.rst]: heh.

File: docs/tutorials/javascript.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Offline and synchronization
+---------------------------
+
+We built :github:`kinto.js <Kinto/kinto.js>`, a JavaScript client dedicated
+to synchronizing remote records with a local database.
+
+The best way to start is :rtd:`to follow the dedicated tutorial <kintojs>` !
+
+Using an offline-first client is not mandatory, see the following section for
+a tutorial using «*vanilla JS*».
```
Comment (almet) [inline docs/tutorials/javascript.rst]: I think the differentiator here isn't that it's vanilla JS, it's more that the data isn't stored locally: this is raw requests to the server.

File: docs/tutorials/javascript.rst [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    fetch(recordsURL, options)
+      .then(function (response) {
+        return response.json();
+      })
+      .then(function (result) {
+        console.log(result.data.length, ' records found.');
+      });
+
+In order to :ref:`poll for changes <api-synchronisation>`, store the last
+synchronization timestamp, and reuse it:
```
Comment (almet) [inline docs/tutorials/javascript.rst]: s ?

File: docs/tutorials/javascript.rst [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+      .then(function (result) {
+        result.data.forEach(function (record) {
+            if (record.deleted) {
+                console.log('Deleted record id=', record.id);
+            }
+            else {
+                console.log('New record id=', record.id);
+            }
+        });
+      });
```
Comment (almet) [inline docs/tutorials/javascript.rst]: @oak11 will like this :)

Comment (almet) [inline docs/tutorials/javascript.rst]: @oak11 will like this :)

File: docs/tutorials/javascript.rst (hunk 3 above)
Comment (leplatrem) [inline docs/tutorials/javascript.rst]: I thought also, but then realize there were inconvenients: 

* the global index is bypassed (and can give some info to get started)
* the link has to be maintained and can be broken more easily

File: docs/tutorials/javascript.rst (hunk 7 above)
Comment (oak11) [inline docs/tutorials/javascript.rst]: yeah!

**Comment by almet** [comment]
go go go !

**Pull request merged**